clang/include/clang/Sema/SemaHLSL.h

```diff
@@ -102,6 +152,15 @@ class SemaHLSL : public SemaBase {
   llvm::DenseMap<const HLSLAttributedResourceType *,
                  HLSLAttributedResourceLocInfo>
       LocsForHLSLAttributedResources;
+
+  // List of all resource bindings
+  ResourceBindings Bindings;
+
+private:
+  void FindResourcesOnVarDecl(VarDecl *D);
+  void FindResourcesOnUserRecordDecl(const VarDecl *VD, const RecordType *RT,
+                                     int Size);
+  void ProcessExplicitBindingsOnDecl(VarDecl *D);
```

Comment:
Better to name these startingWithLowerCase, which LLVM style generally prefers. It's a bit awkward in this area of the code because there's a lot of legacy naming, but if it isn't part of the Sema-interface parts of the class I think it makes sense to use the modern style.

Reply:
I was not sure what is the preferred way as the naming is sometimes all over the place.